cycode/cli/commands/scan/code_scanner.py

```diff
@@ -337,6 +348,7 @@ def scan_commit_range_documents(
                 scan_type,
                 scan_parameters,
                 timeout,
+                should_use_scan_service,
```

Comment:
as i can see the `scan_commit_range_documents` method is used only for SCA (called in `scan_sca_pre_commit` and `scan_sca_commit_range`. SCA always goes to the scan service. i think we don't need here `should_use_scan_service` at all

for Secrets commit history scans `scan_documents` is called with `is_git_diff=True, is_commit_range=True` flags